@inliable initializer can't be designated initializer

when I build this int bazel Xcode 15, I got some errors. they are all same kind error, and this are one of them

```
CoreStore/Sources/Internals.DiffableDataUIDispatcher.DiffResult.swift:256:22: error: initializer for class 'Internals.DiffableDataUIDispatcher<O>.DiffResult<Index>.IndicesReference' is '@inlinable' and must delegate to another initializer
```

--- a/Sources/Internals.DiffableDataUIDispatcher.Changeset.swift
+++ b/Sources/Internals.DiffableDataUIDispatcher.Changeset.swift
@@ -48,7 +48,7 @@ extension Internals.DiffableDataUIDispatcher {
         var elementUpdated: [ElementPath]
         var elementMoved: [(source: ElementPath, target: ElementPath)]
 
-        @inlinable
+        @usableFromInline
         init(
             data: C,
             sectionDeleted: [Int] = [],

--- a/Sources/Internals.DiffableDataUIDispatcher.DiffResult.swift
+++ b/Sources/Internals.DiffableDataUIDispatcher.DiffResult.swift
@@ -190,7 +190,7 @@ extension Internals.DiffableDataUIDispatcher {
         
         // MARK: Private
         
-        @inlinable
+        @usableFromInline
         internal init(
             deleted: [Index] = [],
             inserted: [Index] = [],
@@ -224,7 +224,7 @@ extension Internals.DiffableDataUIDispatcher {
             @usableFromInline
             internal var isTracked = false
             
-            @inlinable
+            @usableFromInline
             init() {}
         }
         
@@ -252,7 +252,7 @@ extension Internals.DiffableDataUIDispatcher {
             @usableFromInline
             internal var position = 0
             
-            @inlinable
+            @usableFromInline
             internal init(_ indices: ContiguousArray<Int>) {
                 
                 self.indices = indices
@@ -292,7 +292,7 @@ extension Internals.DiffableDataUIDispatcher {
             @usableFromInline
             internal let pointer: UnsafePointer<T>
             
-            @inlinable
+            @usableFromInline
             internal init(pointer: UnsafePointer<T>) {
                 
                 self.pointeeHashValue = pointer.pointee.hashValue

--- a/Sources/Internals.DiffableDataUIDispatcher.StagedChangeset.swift
+++ b/Sources/Internals.DiffableDataUIDispatcher.StagedChangeset.swift
@@ -40,7 +40,7 @@ extension Internals.DiffableDataUIDispatcher {
         @usableFromInline
         var changesets: ContiguousArray<Changeset<C>>
 
-        @inlinable
+        @usableFromInline
         init<S: Sequence>(_ changesets: S) where S.Element == Changeset<C> {
             
             self.changesets = ContiguousArray(changesets)

--- a/Sources/Internals.DiffableDataUIDispatcher.swift
+++ b/Sources/Internals.DiffableDataUIDispatcher.swift
@@ -221,7 +221,7 @@ extension Internals {
             @usableFromInline
             var section: Int
 
-            @inlinable
+            @usableFromInline
             init(element: Int, section: Int) {
                 
                 self.element = element

